### ISSUE TITLE
Missing files

### ISSUE THREAD

**Issue by mohan553** [Sam1224/FSSAM#2]
Many thanks to the author for the impressive work, but in the process of reproducing it an error was reported :No such file or directory: ‘priors/pascal/small/dinov2_vitb14/split0/1shot/1.npy’.
Is the file in question not being uploaded?

**Comment by fenghu81** [comment]
我在train.py，做了如下改变，模型就能运行了，不知道对不：
if main_process():
    if not os.path.exists(prior_path):
        os.makedirs(os.path.dirname(prior_path), exist_ok=True)  # 创建目录---new 
        priors = priors.to(torch.float32).detach().cpu().numpy()
        np.save(prior_path, priors)
